src/client/sky.h

```diff
@@ -85,6 +85,7 @@ class Sky : public scene::ISceneNode
 
 	// Set only from set_sky API
 	void setCloudsEnabled(bool clouds_enabled) { m_clouds_enabled = clouds_enabled; }
+	void setStarSeed(u64 star_seed) { m_seed = star_seed; }
```

Comment:
1. I think having this setter here violates the above comment ("set only from set_sky API"); please move it below `setStarDayOpacity`, along with the other star-related setters.
2. More importantly, I think there needs to be a check that calls `updateStars()` if the seed has changed. Otherwise changing the seed during the game does not work. (And whether the seed works at all probably becomes a bit race-condition-ey.)